mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientdetails/ClientDetailsFragment.java

```diff
@@ -599,6 +600,26 @@ public void showUploadImageProgressbar(boolean b) {
         }
     }
 
+
+    public void loadImageGlide(int clientId) {
+        pb_imageProgressBar.setVisibility(VISIBLE);
+        String url = PrefManager.getInstanceUrl()
```

Comment:
For every client detail you are creating a new String URL instance, that too with concatenation which is not a very good idea.

Consider using StringBuilder instead.

Also try to abstract this out into some util and cache the URL with clientID placeholder. So that you only have to insert that later on.

So your cached your would look something like this "blah blah %s blah blah blah"

And now every time you request this url from your Util you have to insert clientId to complete that.